src/Symfony/Component/Console/Tests/Helper/FormatterHelperTest.php

```diff
@@ -39,15 +39,15 @@ public function testFormatBlock()
 
         $this->assertEquals(
             '<error> Some text to display </error>'."\n".
-            '<error> foo bar              </error>',
+                '<error> foo bar              </error>',
             $formatter->formatBlock(['Some text to display', 'foo bar'], 'error'),
             '::formatBlock() formats a message in a block'
         );
 
         $this->assertEquals(
             '<error>                        </error>'."\n".
-            '<error>  Some text to display  </error>'."\n".
-            '<error>                        </error>',
+                '<error>  Some text to display  </error>'."\n".
+                '<error>                        </error>',
```

Comment:
this reads weird now IMHO

Reply:
For me this is absolutely OK and expected. This is one statement, but concatenated, so it's better distinguishable if subsequent lines are indented. Now you clearly see that `$formatter->formatBlock()` (below) is a next statement (argument in this case), while before each line looked like separate argument (`.` is too similar to `,`, it's easy to misinterpret while reading).

Reply:
Yes, I got the intention, in my projects I would just not enable it 😄